README.md

```diff
@@ -50,9 +69,13 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
   @import "bourbon/bourbon";
   ```
 
-  It’s not recommended to add or modify the Bourbon files so that you can update them easily.
+  It’s not recommended that you modify the Bourbon files so that you can update
```

Comment:
Maybe explain will it make the update difficult? Perhaps something like:
 
`It's not recommended that you modify the Bourbon files since it will make updating bourbon difficult. Updating Bourbon will overwrite any custom file changes or cause merge conflicts`